examples/with-app-router-context/next.config.js

```diff
@@ -1,6 +1,7 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   experimental: {
+    serverComponentsExternalPackages: ['next-runtime-env'],
```

Comment:
Should we add a comment here (maybe with a link to the issue where this was mentioned)?